packages/daemon/test/controller/pipeline.test.ts

```diff
@@ -1,4 +1,7 @@
 import { app } from 'midway-mock/bootstrap';
+import { PipelineService } from '../../src/service/pipeline';
+import * as sinon from 'sinon';
+import assert from 'midway-mock/node_modules/@types/power-assert';
```

Comment:
This line should be removed and import `assert` from `midway-mock/bootstrap`.

Reply:
I have fixed at https://github.com/alibaba/pipcook/pull/510/commits/0b630306e3932ead1f0ef39313791de1118bdb26.